.gitignore

```diff
@@ -28,10 +28,10 @@ tmp/
 /apps
 /deploy
 /build
-/compiled
```

Comment:
This seems wrong. Why would you want to commit the `comiled` directory?

Reply:
This is a duplicate entry. There is already a `compiled` in line 27